segway/run.py

```diff
@@ -1444,6 +1463,15 @@ def bedgraph_filename(self):
 
         return self.work_dirpath / basename
 
+    @memoized_property
+    def posterior_bed_filename(self):
```

Comment:
We now have four functions with the same implementation, except for two variable names. Can you please make an underscore method that these all call